### DISCUSSION TITLE
本讨论区内的 docs/ 和 discuss/ 是为了同步 rime.im 的内容吗?

### DISCUSSION THREAD

**Discussion by pdog18** [rime/home#1287] [Q&A]
在 Discussions 中继续把话题分散了, 也不利于以后的用户用关键字检索.

本身就有很多人把 Issues 作为 Discussions 在使用, 现在又一层分裂, 感觉太不合适了. 

建议 Issues 内加上限制, 不能随意发送了, 要让提出 Issue 的用户, 在明确知道不应该在 Discussions 发送的内容才能提 Issue. 

为了以后遇到一些问题, 尽量在一处进行搜索就能明确解决(或明确解决不了)

**Comment by lotem** [comment]
怎么组织讨论内容，这块还在摸索。

docs/ 这些主题是用一个叫giscus的插件，把issues的评论功能内嵌到网站里。
